src/lib.rs

```diff
@@ -524,6 +525,14 @@ impl Config {
         self
     }
 
+    /// Should this report not collapse multiline labels?
+    ///
+    /// If unspecified, this defaults to 'false'
+    pub const fn with_show_full(mut self, show_full: bool) -> Self {
```

Comment:
My feeling is that this function name / comment don't quite describe how this works. I'd suggest `with_show_full_span` and

```
/// Whether all lines covered by a multi-line span are shown, instead of just the first and last lines.
```